experiments/ClimaEarth/run_amip.jl

```diff
@@ -55,7 +55,6 @@ import ClimaUtilities.SpaceVaryingInputs: SpaceVaryingInput
 import ClimaUtilities.TimeVaryingInputs: TimeVaryingInput, evaluate!
 import ClimaUtilities.Utils: period_to_seconds_float
 import ClimaUtilities.ClimaArtifacts: @clima_artifact
-import Interpolations
```

Comment:
Interpolations.jl is used because it triggers an extension loading in ClimaUtilities. We can probably get away with removing it because it is triggered by other dependencies as well, but maybe we should not

Reply:
We should keep it and add a comment noting why we need the import

Reply:
Comment added